### PR TITLE
add Siebel Scholar

### PR DESCRIPTION
**Pull request by moesoha**

### CODE REVIEW

File: RESUME.txt [hunk 1]
```diff
@@ -41,6 +41,8 @@ PUBLICATIONS
     > Yu-Ping Wang, Xu-Qiang Hu, Zi-Xin Zou, * Wende Tan * , Gang Tan. IVT: An Efficient Method for Sharing Subtype Polymorphic Objects. In Proceedings of the ACM on Programming Languages Volume 3 Issue OOPSLA, October 2019.
     > Yu-Ping Wang, * Wende Tan * , Xu-Qiang Hu, Dinesh Manocha, Shi-Min Hu. TZC: Efficient Inter-Process Communication for Robotics Middleware with Partial Serialization. In Proceedings of the 2019 IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS), 2019.
 
+PATENTS
```
Comment (twd2) [inline RESUME.txt]: Hi, could you please remove this empty section manually?

**Review by twd2 — APPROVED** [review]
Thank you :)

File: RESUME.txt (hunk 1 above)
Comment (moesoha) [reply RESUME.txt]: haha, i thought this behavior is expected

Comment (twd2) [reply RESUME.txt]: Oh, my script is full of No.1 code.

**Pull request merged**